src/MemoryTracing/MemoryTracingUtils.cpp

```diff
@@ -85,4 +90,139 @@ std::optional<SystemMemoryUsage> GetSystemMemoryUsage() noexcept {
   return parsing_result;
 }
 
+int64_t ParseProcessStatus(const std::string& status_content) {
```

Comment:
I would consider a different name for this to make it clearer what this function is doing. For example, GetVmRssFromProcessStatus. You can also make it more generic and call it GetValueFromProcessStatus and pass both the status file content and desired field (VmRSS in this case) into the function.

Reply:
Thanks for suggesting the name! Changed to `GetVmRssFromProcessStatus` now. Thanks!